virt/live_migration/virt-initiating-live-migration.adoc

```diff
@@ -0,0 +1,29 @@
+:_content-type: ASSEMBLY
+[id="virt-initiating-live-migration"]
+= Initiating live migration
+include::_attributes/common-attributes.adoc[]
+:context: virt-initiating-live-migration
+
+toc::[]
+
+You can initiate the live migration of a virtual machine (VM) to another node by using the xref:../../virt/live_migration/virt-initiating-live-migration.adoc#virt-initiating-vm-migration-web_virt-initiating-live-migration[{product-title} web console] or the xref:../../virt/live_migration/virt-initiating-live-migration.adoc#virt-initiating-vm-migration-cli_virt-initiating-live-migration[command line].
+
+You can cancel a live migration by using the xref:../../virt/live_migration/virt-initiating-live-migration.adoc#virt-cancelling-vm-migration-web_virt-initiating-live-migration[web console] or the xref:../../virt/live_migration/virt-initiating-live-migration.adoc#virt-cancelling-vm-migration-cli_virt-initiating-live-migration[command line]. The VM remains on its original node.
+
+[TIP]
+====
+You can also initiate and cancel live migration by using the `virtctl migrate <vm_name>` and `virtctl migrate-cancel <vm_name>` commands.
+====
+
+include::modules/virt-initiating-vm-migration-web.adoc[leveloffset=+1]
+
+include::modules/virt-cancelling-vm-migration-web.adoc[leveloffset=+2]
```

Comment:
Re: all instances of "cancelling," the ISG says two consonants for "cancellation" and one for "canceled/canceling." This might be too annoying to fix right now, so maybe we could do it in a follow-up.